### PR TITLE
Add two new resources

### PR DESCRIPTION
**Pull request by kfrn**

### CODE REVIEW

**Review by ablwr — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -4,9 +4,12 @@ In the spirit of [analog-inspection](https://github.com/amiaopensource/analog-in
 
 ### Guidelines and Articles
 
+* **Howard Besser** (2001), [Longevity of Electronic Art
+](http://besser.tsoa.nyu.edu/howard/Papers/elect-art-longevity.html)
 * **Pip Laurensen** (2006), [Authenticity, Change and Loss in the Conservation of Time-Based Media Installations](http://www.tate.org.uk/research/publications/tate-papers/06/authenticity-change-and-loss-conservation-of-time-based-media-installations)
 * **Electronic Arts Intermix** (c.2007), [EAI Online Resource Guide for Exhibiting, Collecting & Preserving Media Art](http://www.eai.org/resourceguide/home.html)
 * **Smithsonian** (May 2010), [Collaborations in Conserving Time-Based Art: A Summary of Discussion Group Sessions of a Colloquium ](https://www.si.edu/content/tbma/documents/Time-Based-Art_Report.pdf)
+* **Dragan Espenschied _et al_** (September 2013), [Large-Scale Curation and Presentation of CD-ROM Art](http://purl.pt/24107/1/iPres2013_PDF/Large-Scale%20Curation%20and%20Presentation%20of%20CD-ROM%20Art.pdf)
```
Comment (retokromer) [inline README.md]: Sorry for being dot-picking: does `et al.` not have a period in English? It is the Latin abbreviation for _et alii_ (masculine), _et aliae_ (feminine) and/or _et alia_ (neuter).

Comment (retokromer) [reply README.md]: Also for the Patrícia Falcão entry.

**Review by retokromer — APPROVED** [review]

**Comment by retokromer** [comment]
Amsterdam thanks you!

File: README.md (hunk 1 above)
Comment (kfrn) [reply README.md]: Yes, it should have a full stop after. Oops!

**Comment by kfrn** [comment]
I'm pleased to hear it! Are you there at the moment @retokromer ?

**Comment by retokromer** [comment]
Yes, with @kieranjol _et al._ for The Reel Thing.

**Comment by kfrn** [comment]
Ha! Awesome. Hopefully I'll make it to one of these events some day. Say hi to KOL and any others for me :smile:

**Pull request merged**